src/otx/core/model/module/segmentation.py

```diff
@@ -126,3 +126,11 @@ def test_step(self, inputs: SegBatchDataEntity, batch_idx: int) -> None:
     def lr_scheduler_monitor_key(self) -> str:
         """Metric name that the learning rate scheduler monitor."""
         return "train/loss"
+
+    def _get_state_dict_from_ckpt(self, ckpt: dict) -> dict:
+        """Get the state_dict, supporting the backward compatibility."""
+        state_dict = super()._get_state_dict_from_ckpt(ckpt)
+        for key in list(state_dict.keys()):
+            if "ham.bases" in key or "decode_head.aggregator.projects" in key:
+                state_dict.pop(key)
+        return state_dict
```

Comment:
@kprokofi , it seems that segmentation models are not compatible with the OTX1.5. Could you check from your side? If you trained the model at the OTX1.5 and then load the checkpoint in the OTX2.0, you might meet an error when you use the `litehrnet_18`.

Although the error log just showed the mis-match of the shape, however, I think there is a gap of model itself between OTX1.5 and OTX2.0. 
![image](https://github.com/openvinotoolkit/training_extensions/assets/80735344/6a5b788e-2f04-4031-a2d3-f00b46a07f8d)